x-pack/test/functional/apps/ml/anomaly_detection/aggregated_scripted_job.ts

```diff
@@ -0,0 +1,472 @@
+          ignore_unavailable: false,
+          allow_no_indices: true,
+          ignore_throttled: true,
+        },
+        query: {
+          bool: {
+            must: [
+              {
+                match_all: {},
+              },
+            ],
+          },
+        },
+        indices: ['ft_ecommerce'],
+        script_fields: {
+          actual_taxed: {
+            script: {
+              source: "doc['taxful_total_price'].value * 1.825",
+              lang: 'painless',
+            },
+            ignore_failure: false,
+          },
+          gender_currency: {
+            script: {
+              source: "doc['customer_gender'].value + '_' + doc['currency'].value",
+              lang: 'painless',
+            },
+            ignore_failure: false,
+          },
+        },
+        scroll_size: 1000,
+        delayed_data_check_config: {
+          enabled: true,
+        },
+        job_id: `fq_unsupported_script_${ts}`,
+        datafeed_id: `datafeed-fq_unsupported_script_${ts}`,
+      } as unknown) as Datafeed,
+    },
+  ];
+
+  describe('aggregated or scripted job', function () {
+    this.tags(['mlqa']);
+    before(async () => {
+      await esArchiver.loadIfNeeded('ml/farequote');
+      await esArchiver.loadIfNeeded('ml/ecommerce');
+      await ml.testResources.createIndexPatternIfNeeded('ft_farequote', '@timestamp');
+      await ml.testResources.createIndexPatternIfNeeded('ft_ecommerce', 'order_date');
+      await ml.testResources.setKibanaTimeZoneToUTC();
+      await ml.securityUI.loginAsMlPowerUser();
+    });
+
+    after(async () => {
+      await ml.api.cleanMlIndices();
+    });
+    for (const testData of supportedTestSuites) {
+      describe(testData.suiteTitle, function () {
+        before(async () => {
+          await ml.api.createAndRunAnomalyDetectionLookbackJob(
+            testData.jobConfig,
+            testData.datafeedConfig
+          );
+        });
+
+        it('opens a job from job list link', async () => {
+          await ml.testExecution.logTestStep('navigate to job list');
+          await ml.navigation.navigateToMl();
+          await ml.navigation.navigateToJobManagement();
+
+          await ml.testExecution.logTestStep(
+            'check that the single viewer metric button is disabled'
+          );
+          await ml.jobTable.waitForJobsToLoad();
+          await ml.jobTable.filterWithSearchString(testData.jobConfig.job_id, 1);
+
+          await ml.jobTable.assertJobActionSingleMetricViewerButtonEnabled(
+            testData.jobConfig.job_id,
+            true
+          );
+        });
+
+        it('displays job results correctly in both anomaly explorer and single metric viewer', async () => {
+          await ml.testExecution.logTestStep('opens job in single metric viewer');
+          await ml.jobTable.clickOpenJobInSingleMetricViewerButton(testData.jobConfig.job_id);
+          await ml.commonUI.waitForMlLoadingIndicatorToDisappear();
```

Comment:
I think the above three lines (to open the job in the SMV) would better fit into the first test block `opens a job from job list link`.

Reply:
Updated here 49899e9